src/rpc/blockchain.cpp

```diff
@@ -953,39 +983,52 @@ static UniValue gettxoutsetinfo(const JSONRPCRequest& request)
 {
             RPCHelpMan{"gettxoutsetinfo",
                 "\nReturns statistics about the unspent transaction output set.\n"
-                "Note this call may take some time.\n",
-                {},
+                "Note this call may take some time if you are not using coinstatsindex.\n",
+                {
+                    {"hash_or_height", RPCArg::Type::NUM, RPCArg::Optional::OMITTED, "The block hash or height of the target height", "", {"", "string or numeric"}},
+                },
                 RPCResult{
                     RPCResult::Type::OBJ, "", "",
                     {
                         {RPCResult::Type::NUM, "height", "The current block height (index)"},
                         {RPCResult::Type::STR_HEX, "bestblock", "The hash of the block at the tip of the chain"},
-                        {RPCResult::Type::NUM, "transactions", "The number of transactions with unspent outputs"},
                         {RPCResult::Type::NUM, "txouts", "The number of unspent transaction outputs"},
                         {RPCResult::Type::NUM, "bogosize", "A meaningless metric for UTXO set size"},
-                        {RPCResult::Type::STR_HEX, "hash_serialized_2", "The serialized hash"},
+                        {RPCResult::Type::STR_HEX, "utxo_set_hash", "The serialized hash"},
```

Comment:
Nit, for when you work on the RPC PR: let's call this `utxo_set_muhash` in case we want to try another algo later.

Reply:
We could additionally expose the full 384-byte hash (base64 encoded).